src/main/java/org/veupathdb/service/eda/ss/model/variable/binary/IdsMap.java

```diff
@@ -0,0 +1,52 @@
+package org.veupathdb.service.eda.ss.model.variable.binary;
+import org.gusdb.fgputil.ListBuilder;
+import org.veupathdb.service.eda.ss.model.variable.VariableValueIdPair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * map of an entity's idIndex to its entity_id and its ancestor's entity_ids
+ * @author sfischer
+ *
+ */
+public class IdsMap extends VariableValueIdPair<List<String>> {
```

Comment:
This name implies a collection of mappings a la java.util.Map.  The Javadoc above seems to confirm this.  A better name might be RecordIdValues, IdsMapEntry, or IdMapping (in my order of preference) :)

Reply:
I agree, renamed in latest